### PR TITLE
Began integration of gpio io class

### PR DESCRIPTION
**Pull request by FredWei-Hub**
Implementation of GPIO class.

### CODE REVIEW

File: projects/HotFire/config/GpioConfig.h [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+    #endif
+
+    #if USE_PWM2
+        #define PWM2_NAME "Valve505"
+        #define PWM2_PIN 14
+        #define PWM2_OPEN 50
+        #define PWM2_CLOSE 0
+    #endif
+
+#endif
```
Comment (ajayyy) [inline projects/HotFire/config/GpioConfig.h:27]: This is nice

File: projects/HotFire/config/config.h [hunk 2]
```diff
@@ -1,15 +1,15 @@
...
+    #define USE_SBG 0
 #endif
 
 #ifndef USE_SOCKET_CLIENT
     #define USE_SOCKET_CLIENT 0
 #endif
 
 #ifndef USE_LOGGER
-    #define USE_LOGGER 1
+    #define USE_LOGGER 0
```
Comment (ajayyy) [inline projects/HotFire/config/config.h:12]: No logs?

Comment (sebastiengrd) [reply projects/HotFire/config/config.h:12]: Oops, changed this for testing. I will put it back to 1

**Review by kovalromank — DISMISSED** [review]

File: src/IO/Gpio.cpp [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+    	//std::cout << output.first << " :: " << output.second << std::endl;
+	}
+	for (std::pair<std::string, int> output : data.outputMap)
+	{
+		outputsMap.at(output.first).setValue(output.second);
+    	//std::cout << output.first << " :: " << output.second << std::endl;
+	}
+
+	//std::cout<< "SetOutputs called with " << data.testValue << "\n";
+}
```
Comment (ajayyy) [inline src/IO/Gpio.cpp:58]: comments

**Review by ajayyy — CHANGES_REQUESTED** [review]

**Review by ajayyy — APPROVED** [review]

**Pull request merged**